certbot-apache/certbot_apache/tests/configurator_test.py

```diff
@@ -1293,5 +1293,10 @@ def test_get_vhost_continue(self, mock_vhost):
         vhs = self.config.get_virtual_hosts()
         self.assertEqual([], vhs)
 
+    def test_wildcard_alias(self):
+        target_name = "banana.com"
+        chosen_vhost = self.config.choose_vhost(target_name)
+        self.assertTrue("*.banana.com" in chosen_vhost.aliases)
```

Comment:
This only tests that we didn't delete the wildcard `ServerAlias`. Really we should be testing that we didn't add one.